lib/srv/discovery/discovery.go

```diff
@@ -225,7 +231,7 @@ kubernetes matchers are present.`)
 	}
 
 	if c.PollInterval == 0 {
```

Comment:
can we change the places where this var is used and convert the time.Ticker to time.Timer instead?
Otherwise people will start reducing it so much that the polling time will be higher than the tick pacer

Reply:
https://github.com/gravitational/teleport/pull/41768